spec/controllers/assessment360_controller_spec.rb

```diff
@@ -161,55 +161,6 @@
         returned_meta_review = controller.instance_variable_get(:@meta_review)
         expect(returned_meta_review[nil][1]).to eq("100%")
       end
-    end
-  end
-
-  describe '#course_student_grade_summary' do
```

Comment:
please remove all instances of course_student_grade_summary as we won't be using it.